sky/packages/sky/lib/gestures/arena.dart

```diff
@@ -32,6 +32,8 @@ class GestureArenaEntry {
 class GestureArena {
```

Comment:
Document in the READMEs what the gestures/ directory's role is in all the dependency layering.